.github/workflows/deployment.yaml

```diff
@@ -13,20 +13,28 @@ jobs:
         with:
           node-version: '16.x'
           cache: 'yarn'
+      - run: |
+          if [ -e yarn.lock ]; then
+            yarn install --frozen-lockfile
+          elif [ -e package-lock.json ]; then
+            npm ci
+          else
+            npm i
+          fi
       - uses: webfactory/ssh-agent@v0.5.3
         with:
           ssh-private-key: ${{ secrets.GH_PAGES_DEPLOY }}
+      - name: Generate Seneca-IPC144.pdf
+        run: |
+          chmod +x ./convert-to-pdf.sh
+          git add .
```

Comment:
It's here @humphd

Reply:
I don't think I'm being clear enough. What I want you to do is remove line 29.  Instead, I want you to run that line in your bash shell and then `git add ./convert-to-pdf.sh` and commit.  Git will record the executable change to that file.  Then push this as part of this branch.

Reply:
> I don't think I'm being clear enough. What I want you to do is remove line 29. Instead, I want you to run that line in your bash shell and then `git add ./convert-to-pdf.sh` and commit. Git will record the executable change to that file. Then push this as part of this branch.

@humphd  thanks for your explanation, I had tried changing the file's executable locally and then `git status` but it didn't show any differences so I thought I had to  `git add` and `commit` in the `.yaml` to maintain the executable status to that file. But yeah, I removed the adding and committing in the `.yaml` as you requested